content/tutorials/get-ether-and-ocean-tokens.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Get Ether and Ocean Tokens for Testnets
+description: A tutorial about how to get Ether and Ocean Tokens for testnets.
+---
+
+If you want to interact with a testnet, then you'll eventually need Ether or Ocean Tokens _for that testnet_. (Every Ethereum-based network has its own Ether and its own Ocean Tokens, and you can't use those in other networks, or at least it wasn't possible when we wrote this.)
+
+At the time of writing, there were two public testnets you could use to test an Ocean Protocol application: the Kovan Testnet and the Ocean Testnet. For more information about those, see the page about [testnets](/concepts/testnets/). You could also run a local testnet (i.e. on your local machine).
+
+## Get a Compatible Wallet
+
+You will need a wallet that can hold Ether (for any Ethereum network) and Ocean Tokens (for any Ethereum network). For now, we recommend using [MetaMask](https://metamask.io/).
+
+- Follow the MetaMask instructions to install it on your machine.
+- In MetaMask, switch from the **Main Ethereum Network** to the **Kovan Test Network** or the test network you're using.
+
+## Get Ether
+
+### Get Ether for the Kovan Testnet
+
+You can get Kovan Ether (KEth), for the Kovan Testnet, from a Kovan faucet: see [the official list of Kovan faucets](https://github.com/kovan-testnet/faucet). You have to give the faucet your Kovan address (wallet account address). You can get that from MetaMask. It's a string that looks like:
+
+```text
+0xa0A9d7f78bF293514e7cA2789A0Af689eEC99282
+```
+
+### Get Ether for the Ocean Testnet
+
+At the time of writing, there was no easy way to get Ether for the Ocean Testnet.
+
+### Get Ether for a Local Testnet
+
+If you're running a local testnet, then it creates several accounts at network launch time, and gives each of them some Ether. The addresses and private keys of those accounts should be shared (to logs or the console) during the launch process. You can use those accounts and their Ether.
+
```

Comment:
Upper line is true if user is running ganache node from (default) docker-images. If he is running docker-images with local parity node (--local-parity-node and there is no accounts listed on startup) then he can send himself Ether with:
Running local parity node from docker-images you can give yourself tokens with a command line
`curl --data '{"jsonrpc":"2.0","method":"personal_sendTransaction","params":[{"from":"0x00Bd138aBD70e2F00903268F3Db08f2D25677C9e","to":"<YOUR ADDRESS>","value":"0x7FFFFFFFFFFFFFFFFFF"}, "node0"],"id":0}' -H "Content-Type: application/json" -X POST localhost:8545` or directly creating new account on parity node with ?command?

Reply:
Thanks, I'll add that.